### ISSUE TITLE
Library versions

### ISSUE THREAD

**Issue by ogencoglu** [sweta20/Detecting-Cyberbullying-Across-SMPs#2]
What are the versions for the used libraries? A _requirements.txt_ file with exact versions would be great for reproducibility.

**Comment by sachinksachu** [comment]
please add versions of libraries used. Thanks in advance !!

**Comment by sweta20** [comment]
I can check and update. Meanwhile, this is from a [reproducibility ](https://github.com/cmry/amica/tree/e2cd0503495ac34e6c7502ce635e65c6e81b54f6) study:
# base
scikit-learn==0.21.3
pandas==0.25.2
spacy==2.2.1
tqdm==4.36.1
numpy=1.16.4

# optional
keras==2.3.1            # (for reproducing neural models)
tensorflow==1.13.1      # same, backend to keras
tflearn==0.3.2          # for padding etc. in reproduction

**Comment by akrlowicz** [comment]
Hello,
as above mentioned libraries are for python 3 and the project was done in python 2.7 I would still like to know which versions were used :)
